x-pack/plugins/security_solution/public/resolver/data_access_layer/factory.ts

```diff
@@ -50,7 +52,7 @@ export function dataAccessLayerFactory(
       after?: string
     ): Promise<ResolverPaginatedEvents> {
       return context.services.http.post('/api/endpoint/resolver/events', {
-        query: { afterEvent: after },
+        query: { afterEvent: after, limit: relatedEventsPaginationSize },
```

Comment:
Maybe this should be specified on the BE as the 'default' instead of specifying it here. That way we don't have two places defining this value.